content/en/blog/2023/end-user-discussions-04.md

```diff
@@ -0,0 +1,111 @@
+---
+title: OpenTelemetry End-User Discussions Summary for April 2023
+linkTitle: End-User Discussions April 2023
+date: 2023-05-02
+author: '[Pranay Prateek](https://github.com/pranay01) (SigNoz)'
+---
+
+For the month of April 2023, the OpenTelemetry end-user group meet took place
+for users in the Asia-Pacific (APAC) region.
+
+The discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#2f83c1c1-918c-4c78-8671-194b2e9d8e54),
+and everyone in attendance votes on what they want to talk about.
+
+## What we talked about
+
+We talked about evangelizing the adoption of OpenTelemetry in a big organization
+and also discussed how to optimize observability data at scale.
+
+## Discussion Highlights
+
+Below is the summary of this month's discussion.
+
+### Evangelizing adoption of OpenTelemetry in a big organization
+
+**Q:** How do you evangelize the adoption of OpenTelemetry in a big
+organization?
+
+**A:** In a big organization, the first step would be to put out the current
+pain points of observability to leadership. There are benefits of having an open
+source standard for observability. If there is no standard in place, it gets
+very difficult to communicate across different teams. If you use OpenTelemetry,
+you do not have to depend on any vendor agents, and you have the flexibility to
+send data to multiple backends.
+
+### How to optimize observability data at scale?
+
+**Q:** In a big organization, observability data can be in the range of TBs per
+day, which comes with associated costs. But there is always a feeling that 80%
+of captured data is unusable. None of the vendors help you understand what data
+is accessed and how to bring that visibility to the engineering teams sending
+the data.
+
+**A:** One of the ways to optimize observability at scale is sampling. There are
+a number of options for you to reduce the data volumes at the SDKs level and the
+collector level.
+
+Also, there is active work going on the OpenTelemetry Collector side to handle
+data at scale more efficiently. For example, there is work going around using
+Apache Arrow for serialization to optimize network costs.
```

Comment:
can we provide links to that, e.g. to the OTEP on apache arrow so people can follow that conversation and see where we are standing